models/comments_models.js

```diff
@@ -0,0 +1,22 @@
+const { log } = require("console");
+const db = require("../db/connection")
+const articles = require("../db/data/test-data/articles")
+
+exports.selectCommentsByArticleId = async (article_id) => {
+    const result = await db.query(
+        `SELECT comment_id, votes, created_at, author, body, article_id FROM comments
+        WHERE article_id = $1
+        ORDER BY created_at DESC`,
+        [article_id]
+    );
+    if (result.rows.length === 0 && article_id <= articles.length) {
+        return Promise.reject({ status: 200, msg: 'No comments found for this article' })
```

Comment:
If I wanted to get all the comments for an article with comments, I'd expect to get back an array filled with comment objects
If I wanted to get all the comments for an article with no comments, I'd still expect an array, but it would be empty
🔍